detox/e2e/support/test_config.ts

```diff
@@ -1,14 +1,14 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-export const serverOneUrl = process.env.SITE_1_URL || (process.env.IOS === 'true' ? 'http://127.0.0.1:8065' : 'http://10.0.2.2:8065');
-export const siteOneUrl = process.env.SITE_1_URL || 'http://127.0.0.1:8065';
-export const serverTwoUrl = process.env.SITE_2_URL || 'https://mobile02.test.mattermost.cloud';
-export const siteTwoUrl = process.env.SITE_2_URL || 'https://mobile02.test.mattermost.cloud';
-export const serverThreeUrl = process.env.SITE_3_URL || 'https://mobile03.test.mattermost.cloud';
-export const siteThreeUrl = process.env.SITE_3_URL || 'https://mobile03.test.mattermost.cloud';
+export const serverOneUrl = process.env.SITE_1_URL || (process.env.IOS === 'true' ? 'http://localhost:8065' : 'http://10.0.2.2:8065');
+export const siteOneUrl = process.env.SITE_1_URL || 'http://localhost:8065';
+export const serverTwoUrl = process.env.SITE_2_URL || 'http://localhost:8065';
+export const siteTwoUrl = process.env.SITE_2_URL || 'http://localhost:8065';
+export const serverThreeUrl = process.env.SITE_3_URL || 'http://localhost:8065';
+export const siteThreeUrl = process.env.SITE_3_URL || 'http://localhost:8065';
 export const smtpUrl = process.env.SMTP_URL || 'http://127.0.0.1:9001';
-export const adminEmail = process.env.ADMIN_EMAIL || 'sysadmin@sample.mattermost.com';
+export const adminEmail = process.env.ADMIN_EMAIL || 'success+sysadmin@simulator.amazonses.com';
```

Comment:
Since we are using local dev for default setup. I think this should match the ones auto-generated from local. Should this be reverted to `sysadmin@sample.mattermost.com`?